src/faker/provider/internet/internet_fa_IR.nim

```diff
@@ -0,0 +1,22 @@
+include interfaces
+import strutils
+import ../person/person_fa_IR
+
+proc pickFinglish(f: Faker): string =
+  f.rand.sample(finglish) & f.rand.sample(finglish) & (
+    if rand(1..5) < 5: $rand(1..9999)
+    else: ""
```

Comment:
hmm...?
Compile error was occured in Nim 1.0.x, and 1.2.x.

Please check this [CI Log](https://github.com/jiro4989/faker/runs/4782151899?check_suite_focus=true#step:4:10) and fix it.